lua.nanorc

```diff
@@ -65,7 +67,7 @@ color red "\"(\\.|[^\\\"])*\"|'(\\.|[^\\'])*'"
 color red start="\s*\[\[" end="\]\]"
 
 # Escapes
-color red "\\[0-7][0-7][0-7]|\\x[0-9a-fA-F][0-9a-fA-F]|\\[abefnrs]|(\\c|\\C-|\\M-|\\M-\\C-)."
+#color red "\\[0-7][0-7][0-7]|\\x[0-9a-fA-F][0-9a-fA-F]|\\[abefnrs]|(\\c|\\C-|\\M-|\\M-\\C-)."
```

Comment:
Any reason you've removed the escapes highlighting?

Reply:
Opps, I migrated the original author of this fix without looking into this line. It should be remained plus the character set `[abefnrs]` should also be updated.